docs/app/pages/components/components-sections/search/toolbar-search/toolbar-search.component.html

```diff
@@ -68,8 +68,11 @@
 
 <p>
     Other components can be added as required. The toolbar search component on the left above shows how to include an
-    optional "clear" button to the input field using the built-in <code>ux-toolbar-search-clear</code> class. Another
-    common use-case is to include a <a href="https://valor-software.com/ngx-bootstrap/#/typeahead">typeahead</a> list to
+    optional "clear" button to the input field using the built-in <code>ux-toolbar-search-clear</code> class.
+    If the clear button is added, you should add <code>[class.ux-toolbar-search-clear-offset]="clickEvent"</code> to the
+    <code>Input</code> field element (with <code>"clickEvent"</code> being the name of the click event variable).
```

Comment:
Because we expect people to be familiar with most Angular syntax and functionality, I think you can just list the name of the class here, and refer to the example to show how to use it. E.g.

"If the clear button is added, you should add the <code>ux-toolbar-search-clear-offset</code> to the <code>input</code> field element to apply the appropriate padding. The sample code shows an example of binding this class."

Reply:
No worries, wasn't sure exactly how simplified/well explained it needed to be.